src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -371,6 +364,13 @@ public sealed class ConvertToXmlCommand : PSCmdlet, IDisposable
     {
         #region Command Line Parameters
 
+        /// <summary>
+        /// 
+        /// </summary>
+        [Parameter()]
+        [ValidateSet("CliXml")]
+        public string Format { get; set; }
```

Comment:
This should probably be an enum value, to make it simpler to add items to in future if other XML formats are desired. Both `CliXml` and `Normal` values should be allowed (`Normal` being the default setting)